### DISCUSSION TITLE
Anonimizzazione di tutti i dati del segnalante

### DISCUSSION THREAD

**Discussion by marcopvl** [globaleaks/globaleaks-whistleblowing-software#4302] [General]
Buongiorno, sapete dirmi per cortesia se esiste una versione di GL, oppure una feature che consenta di attivare l'anonimizzazione di tutti i dati del segnalante ?
Ovvero rendere anonimi tutti i dati del segnalante, di modo che nessuno possa sapere la sua identità.
Grazie mille

**Comment by evilaliv3** [comment]
Buongiorno @marcopvl , grazie per la domanda.

A quali dati del segnalante ti riferisci?

La piattaforma di default anonimizza i metadati di connessione (e.g. non registra indirizzi ip e user agent del segnalante) e laddove il segnalante utilizzi il Tor Browser la stessa neppure li riceve.

**Comment by marcopvl** [comment]
Ciao @evilaliv3 , mi riferisco ai dati del segnalante (Nome, Cognome ed indirizzo Email) che arrivano al ricevente.

**Comment by evilaliv3** [comment]
Se nessuno deve conoscere la sua identità in modo assoluto potete configurare il questionario per non raccoglierla rimuovendo tali domande.

In alternativa se avete volonta il nome, il cognome e l'indirizzo email vengano raccolti ma non siano eventualmente solo visibili su necessità potete mettere a sistema un utente Custode. Questi potrà vagliare le richieste dei riceventi di accedere a tali dati.

**Comment by marcopvl** [comment]
Grazie @evilaliv3 facciamo qualche prova con il Custode e vediamo se funziona come ci aspettiamo. 
Grazie ancora

**Comment by marcopvl** [comment]
Ciao @evilaliv3 segnalo che nella nostra installazione in italiano, non riusciamo a trovare il campo "whistleblower identity" per attivare la domanda per il custode.
In allegato l'immagine dove non è presente tale tipologia di domanda:

![immagine](https://github.com/user-attachments/assets/e6e4f747-ac39-4ed2-aa4c-b2533067fdcf)

Potresti farmi sapere per cortesia se sbagliamo qualcosa o come attivarla ?
